devel/reader-sync/solana-validator-DihkpHCvJoJS91WDRiHtkVHkwotuAhDTHq5d3tREeRej.log

```diff
@@ -7,10 +7,10 @@
             "--limit-ledger-size",
```

Comment:
I doubt this was suppose to be committed, @jubeless can you tell if this file should be simply removed from Git?

Reply:
@maoueh that is correct. Should have added it to gitignore